test/functional/test_framework/.mintlayer/config.toml

```diff
@@ -9,5 +9,8 @@ bind_address = "/ip6/::1/tcp/3031"
 ban_threshold = 100
 outbound_connection_timeout = 10
 
+[p2p.mdns_config]
+type = "Disabled"
```

Comment:
Is there a way to change the `type` to something else? I wonder. I think that's the default for enums, but there probably is a way to name it whatever we want.

Reply:
Perhaps `serde(untagged)` can allow to write something like `p2p.mdns_config = Disabled`.

Reply:
`type` can be changed to something else but at least I couldn't get `untagged` to work after playing with it briefly.

Reply:
Renamed it to `state`